### PR TITLE
Added Set-NsxLogicalSwitch cmdlet

### PR DESCRIPTION
**Pull request by kanecharles92**
Added functionality to set values on a NSX Logical Switch object

### CODE REVIEW

File: module/PowerNSX.psm1 [hunk 1]
```diff
@@ -9923,6 +9923,7 @@ function Get-NsxLogicalSwitch {
     }
 }
 
+
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: remove this return line ?

File: module/PowerNSX.psm1 [hunk 2]
```diff
@@ -10004,6 +10005,95 @@ function New-NsxLogicalSwitch  {
...
+    An NSX Logical Switch provides L2 connectivity to VMs attached to it.
+    A Logical Switch is 'bound' to a Transport Zone, and only hosts that are
+    members of the Transport Zone are able to host VMs connected to a Logical
+    Switch that is bound to it.  All Logical Switch operations require a
+    Transport Zone.
+
+    .EXAMPLE
+    Change the name on an existing Logical Switch (LS1 to LS2)
+
+    PS C:\> $ls = Get-NsxLogicalSwitch -name LS1 | Set-NsxLogicalSwitch -name LS2
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: set command before description

add also more example ? (for tenant, description...)

File: module/PowerNSX.psm1 [hunk 3]
```diff
@@ -10080,6 +10170,7 @@ function Remove-NsxLogicalSwitch {
     end {}
 }
 
+
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: remove return line

File: module/PowerNSX.psm1 [hunk 4]
```diff
@@ -15548,6 +15639,60 @@ function Remove-NsxEdgeCertificate{
     end {}
 }
 
+#########
+#########
+# Edge DNS functions
+
+function Get-NsxEdgeDNS {
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: for another i think ?

i have already write code for this
https://github.com/vmware/powernsx/pull/426 (but no yet merged...)

Comment (alagoutte) [reply module/PowerNSX.psm1]: Sure ! no problem ;-) (it is also for avoid to work on the same stuff..)

**Comment by alagoutte** [comment]
Oups, i write more or less the same patch #587 (for #586) (and it is merged) but missing ControlPlaneMode (but it will be easy to add)

@kanecharles92, do you need ControlPlaneMode parameter ?

**Comment by dcoghlan** [comment]
closing this as Set-NsxLogicalSwitch already exists (albeit without ControlPlaneMode). If the ability to change control plane mode of an existing logical switch is required, please re-base and re-submit.

**Pull request closed without merging**